code/modules/mob/living/basic/lavaland/raptor/raptor_ai.dm

```diff
@@ -0,0 +1,181 @@
+#define NEXT_EAT_COOLDOWN 45 SECONDS
```

Comment:
make a new folder in `basic/lavaland/raptor` for all of the AI stuff and then have one file for the controllers, one file for the strategies, and another file for the targeting strategy. this is incredibly hard to track otherwise.